tests/integration/test_recipe.py

```diff
@@ -1289,6 +1289,18 @@ def test_diagnostic_task_provenance(
     assert os.path.exists(prefix + '.svg')
 
 
+def _test_bibtex_files(product_tags):
```

Comment:
This test should be implemented in the ESMValTool repository instead. It should read all the yaml files in the `esmvaltool/recipes` directory and check that there is a bibtex file available for each reference in the 'references' list in the `documentation` section of the recipes. I think we discussed this, my apologies if that was not clear.

Reply:
Could be in `unit/test_recipes.py`, function called `test_bibtex_available` or something similar

Reply:
The test is found here: 
https://github.com/ESMValGroup/ESMValTool/pull/1545/files?file-filters%5B%5D=.py#diff-2266a659a150d71d8451a1568a6948a9